external/quarkus_quickstarts/README.md

```diff
@@ -0,0 +1,69 @@
+- Copy results from Docker container for easier viewing and triage in Jenkins 
+- Quick compare view, easy comparison of how different implementations stack up
+- Parallel testing (to improve execution time)
+- Startup-only testing (application startup, but not full runs of app functional testing)
+- Add high-value tests that exercise the AdoptOpenJDK binaries, including but not limited to functional test suites and Microprofile compliance tests (plan to start with [Fault Tolerance TCK](https://github.com/eclipse/microprofile-fault-tolerance/blob/master/tck/running_the_tck.asciidoc) and [Metrics API TCKs](https://github.com/eclipse/microprofile-metrics/blob/master/tck/running_the_tck.asciidoc) against [GlassFish](https://javaee.github.io/glassfish/) EE reference implementation) 
+ 
+#### Strategic Goals
+- Engage with application communities, including the Eclipse Jakarta EE project, to:
+    - report and resolve application test failures
+    - get more involvement with testing AdoptOpenJDK binaries
+    - encourage use of AdoptOpenJDK binaries, add extra OpenJDK versions and variants in their build farms
+
+### Triage Rules
+There are 4 common triage scenarios, with associated appropriate actions to take:
+![3rd Party App Test Triage Scenarios](../doc/diagrams/appTestTriageScenarios.png)
+
+### How to Add New Tests
+- Learn how to run the application tests that you intend to automate in the build manually first, and find out any special dependencies the application testing may have.
+- Clone https://github.com/AdoptOpenJDK/openjdk-tests.git and look at external directory.
+- Copy the 'example-test' subdirectory and rename it after the application you are adding. 
+- Modify the files in your new sub-directory according to your needs. 
+- Check in the changes into https://github.com/[YOUR-BRANCH]/openjdk-tests and test it using a <a href="https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Personal-Test-Build-on-Jenkins">personal build</a>. 
+
+#### Which files do I need to modify after making a copy of example-test?
+
+**Dockerfile**
+- The example Dockerfile contains a default list of dependent executable files. Please read the documentation of the third party application you are enabling to find out if you need any executable files other than the default set, if yes, add them to the list.   
+- Update the clone command based on your third party application's source repository.
+ 
+ **Shell script**
+- Replace the example command line at the bottom of this script with the initial command lines that trigger execution of your test.
+
+**build.xml** 
+- Update the distribution folder paths, docker image name etc according to the name of your application. 
+
+**playlist.xml** 
+- Update the name of the example test case to the actual test case of the third party application that you intend to run. 
+
+Please direct questions to the [#testing Slack channel](https://adoptopenjdk.slack.com/messages/C5219G28G).
```

Comment:
Can remove these lines, we just wanted to include instructions for running the specific tests in this quarkus_quickstarts subfolder